src/appleseed.python/TXConverter.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+class TXConverter:
+    def __init__(self, maketx_path):
+        self.converted = {}
+        self.maketx_path = maketx_path
+
+    def convert(self, path):
+        if path in self.converted:
+            return self.converted[path]
+        else:
+            path_converted = self._convert(path)
+            self.converted[path] = path_converted
+            return path_converted
+
+    def _convert(self, path):
+        tx_path = self._change_ext_to_tx(path)
+        if os.path.exists(tx_path):
+            print('Warning: {tx_texture} exists.'.format(tx_texture=tx_path))
```

Comment:
after changing prints on logging module logs aren't printed to stdout. Do you have any thought why?